### PR TITLE
Octopus internal review

### PR DESCRIPTION
**Pull request by FPlohl**
Add support for Norik Octopus SoM & IO-Board (nRF9160).

### CODE REVIEW

File: boards/norik/octopus_io_board/Kconfig.defconfig [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Copyright (c) 2023 Nordic Semiconductor ASA
```
Comment (pfiser) [inline boards/norik/octopus_io_board/Kconfig.defconfig]: Can we go with:
Copyright (c) 2024 Norik systems d.o.o.
here?

File: boards/norik/octopus_io_board/Kconfig.norik_octopus_io_board [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# Copyright (c) 2024 Norik Systems
```
Comment (pfiser) [inline boards/norik/octopus_io_board/Kconfig.norik_octopus_io_board]: Can we go with:
Copyright (c) 2024 Norik systems d.o.o.
here?

File: boards/norik/octopus_io_board/board.cmake [hunk 3]
```diff
@@ -0,0 +1,6 @@
+# Copyright (c) 2023 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+board_runner_args(jlink "--device=nRF9160_xxAA" "--speed=4000")
+include(${ZEPHYR_BASE}/boards/common/nrfjprog.board.cmake)
+include(${ZEPHYR_BASE}/boards/common/jlink.board.cmake)
```
Comment (pfiser) [inline boards/norik/octopus_io_board/board.cmake]: Can we add new-line here?

File: boards/norik/octopus_io_board/board.cmake [hunk 4]
```diff
@@ -0,0 +1,6 @@
+# Copyright (c) 2023 Nordic Semiconductor ASA
```
Comment (pfiser) [inline boards/norik/octopus_io_board/board.cmake]: Can we go with:
Copyright (c) 2024 Norik systems d.o.o.
here?

File: boards/norik/octopus_io_board/doc/img/Norik_Octopus_IO-Board_block_diagram.png
Comment (pfiser) [inline boards/norik/octopus_io_board/doc/img/Norik_Octopus_IO-Board_block_diagram.png:1]: Can you move "Octopus SoM" label into the SoM block?

Same goes for GPS uFL and LTE uFL

File: boards/norik/octopus_io_board/doc/index.rst [hunk 5]
```diff
@@ -0,0 +1,154 @@
...
+Octopus IO Board offers the following features:
+
+* Battery charger
+* USB-C for power
+* Solar charger
+* Alkaline battery input
+* LDO regulator to power Octopus SoM and peripherals
+* Battery monitoring using ADC
+* 64 Mbit SPI NOR flash
+* Dedicated ADC, GPIO, I2C, SPI and UARTE pins for expansion
```
Comment (pfiser) [inline boards/norik/octopus_io_board/doc/index.rst:47]: UARTE? Or just UART?

File: boards/norik/octopus_io_board/doc/index.rst [hunk 6]
```diff
@@ -0,0 +1,154 @@
...
+* Exposed headers for current measurements
+* Nano SIM card connector
```
Comment (pfiser) [inline boards/norik/octopus_io_board/doc/index.rst]: drop card, just Nano SIM connector?

File: boards/norik/octopus_io_board/norik_octopus_io_board_common-pinctrl.dtsi [hunk 7]
```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor
```
Comment (pfiser) [inline boards/norik/octopus_io_board/norik_octopus_io_board_common-pinctrl.dtsi]: Can we go with:
Copyright (c) 2024 Norik systems d.o.o.
here?

File: boards/norik/octopus_io_board/norik_octopus_io_board_common-pinctrl.dtsi [hunk 8]
```diff
@@ -0,0 +1,67 @@
...
+	};
+
+	uart0_sleep: uart0_sleep {
+		group1 {
+			psels = <NRF_PSEL(UART_TX, 0, 23)>,
+				<NRF_PSEL(UART_RX, 0, 23)>;
+			low-power-enable;
+		};
+	};
+    i2c0_default: i2c0_default {
```
Comment (pfiser) [inline boards/norik/octopus_io_board/norik_octopus_io_board_common-pinctrl.dtsi]: identation tabs vs spaces?

File: boards/norik/octopus_io_board/norik_octopus_io_board_common-pinctrl.dtsi [hunk 9]
```diff
@@ -0,0 +1,67 @@
...
+	};
+
+	i2c0_sleep: i2c0_sleep {
+		group1 {
+			psels = <NRF_PSEL(TWIM_SDA, 0, 4)>,
+					<NRF_PSEL(TWIM_SCL, 0, 3)>;
+			low-power-enable;
+		};
+	};
+    i2c1_default: i2c1_default {
```
Comment (pfiser) [inline boards/norik/octopus_io_board/norik_octopus_io_board_common-pinctrl.dtsi]: identation tabs vs spaces?

File: boards/norik/octopus_io_board/norik_octopus_io_board_common-pinctrl.dtsi [hunk 10]
```diff
@@ -0,0 +1,67 @@
...
+	spi3_sleep: spi3_sleep {
+		group1 {
+			psels = <NRF_PSEL(SPIM_SCK, 0, 10)>,
+				<NRF_PSEL(SPIM_MISO, 0, 8)>,
+				<NRF_PSEL(SPIM_MOSI, 0, 9)>;
+			low-power-enable;
+		};
+	};
+
+};
```
Comment (pfiser) [inline boards/norik/octopus_io_board/norik_octopus_io_board_common-pinctrl.dtsi]: add new line at the end

File: boards/norik/octopus_io_board/norik_octopus_io_board_common.dtsi [hunk 11]
```diff
@@ -0,0 +1,182 @@
+/*
+ * Copyright (c) 2023-2024 Norik Systems
```
Comment (pfiser) [inline boards/norik/octopus_io_board/norik_octopus_io_board_common.dtsi]: Can we go with:
Copyright (c) 2024 Norik systems d.o.o.
here?

File: boards/norik/octopus_io_board/norik_octopus_io_board_common.dtsi [hunk 12]
```diff
@@ -0,0 +1,182 @@
...
+	   zephyr,shell-uart = &uart0;
+	   zephyr,uart-mcumgr = &uart0;
+   	};
+
+   	leds {
+	   compatible = "gpio-leds";
+	   led0: led_0 {
+		   gpios = <&gpio0 7 0>;
+		   label = "Green LED 1";
+	   };
```
Comment (pfiser) [inline boards/norik/octopus_io_board/norik_octopus_io_board_common.dtsi]: identation?

File: boards/norik/octopus_io_board/norik_octopus_io_board_common.dtsi [hunk 13]
```diff
@@ -0,0 +1,182 @@
...
+
+	simselect-controller{
+		compatible = "gpio-keys";
+		sim_select: sim_select {
+			label = "SIM select";
+			gpios = <&gpio0 25 GPIO_ACTIVE_HIGH>;
+		};
+	};
+
+   /* These aliases are provided for compatibility with samples */
```
Comment (pfiser) [inline boards/norik/octopus_io_board/norik_octopus_io_board_common.dtsi]: Identation (whole block)?

File: boards/norik/octopus_io_board/norik_octopus_io_board_common.dtsi [hunk 14]
```diff
@@ -0,0 +1,182 @@
...
+
+
+	pinctrl-0 = <&spi3_default>;
+	pinctrl-1 = <&spi3_sleep>;
+	pinctrl-names = "default", "sleep";
+	mc3635: mc3635@0 {
+		compatible = "vnd,spi-device";
+		spi-max-frequency = <1000000>;
+		reg = <0>;
+//		int1-gpios = <&gpio0 12 0>;
```
Comment (pfiser) [inline boards/norik/octopus_io_board/norik_octopus_io_board_common.dtsi]: Drop int1-gpios if not used?

File: boards/norik/octopus_io_board/norik_octopus_io_board_common.dtsi [hunk 15]
```diff
@@ -0,0 +1,182 @@
...
+            status = "okay";
+            label = "BQ25180";
+            reg = <0x6a>;
+    };
+};
+
+&spi3 {
+	compatible = "nordic,nrf-spim";
+	status = "okay";
+
```
Comment (pfiser) [inline boards/norik/octopus_io_board/norik_octopus_io_board_common.dtsi]: Why space above and bellow?

File: boards/norik/octopus_io_board/norik_octopus_io_board_common.dtsi [hunk 16]
```diff
@@ -0,0 +1,182 @@
...
+	pinctrl-1 = <&spi3_sleep>;
+	pinctrl-names = "default", "sleep";
+	mc3635: mc3635@0 {
+		compatible = "vnd,spi-device";
+		spi-max-frequency = <1000000>;
+		reg = <0>;
+//		int1-gpios = <&gpio0 12 0>;
+		label = "mc3635";
+	};
+	w25q64: w25q64@0 {
```
Comment (pfiser) [inline boards/norik/octopus_io_board/norik_octopus_io_board_common.dtsi]: add space above?

File: boards/norik/octopus_io_board/norik_octopus_io_board_common.dtsi [hunk 17]
```diff
@@ -0,0 +1,182 @@
...
+		};
+		storage_partition: partition@f8000 {
+			label = "storage";
+			reg = <0x000f8000 0x00008000>;
+		};
+	};
+};
+
+/ {
+
```
Comment (pfiser) [inline boards/norik/octopus_io_board/norik_octopus_io_board_common.dtsi]: Remove extra space?

File: boards/norik/octopus_io_board/norik_octopus_io_board_defconfig [hunk 18]
```diff
@@ -0,0 +1,20 @@
+# SPDX-License-Identifier: Apache-2.0
```
Comment (pfiser) [inline boards/norik/octopus_io_board/norik_octopus_io_board_defconfig]: Do we need to add copyright here?

File: boards/norik/octopus_io_board/norik_octopus_io_board_nrf9160.dts [hunk 19]
```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) 2023-2024 Norik Systems
```
Comment (pfiser) [inline boards/norik/octopus_io_board/norik_octopus_io_board_nrf9160.dts]: Can we go with:
Copyright (c) 2024 Norik systems d.o.o.
here?

File: boards/norik/octopus_io_board/norik_octopus_io_board_nrf9160.yaml [hunk 20]
```diff
@@ -0,0 +1,17 @@
+identifier: norik_octopus_io_boardnrf9160
```
Comment (pfiser) [inline boards/norik/octopus_io_board/norik_octopus_io_board_nrf9160.yaml]: Is it boardnrf9160 or board_nrf9160?

File: boards/norik/octopus_io_board/norik_octopus_io_board_nrf9160.yaml [hunk 21]
```diff
@@ -0,0 +1,17 @@
...
+  - zephyr
+  - gnuarmemb
+  - xtools
+supported:
+  - gpio
+  - i2c
+  - spi
+  - pwm
+  - watchdog
+vendor: norik
```
Comment (pfiser) [inline boards/norik/octopus_io_board/norik_octopus_io_board_nrf9160.yaml]: Add new line at the end?

File: boards/norik/octopus_io_board/norik_octopus_io_board_nrf9160_ns.dts [hunk 22]
```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2023-2024 Norik Systems
```
Comment (pfiser) [inline boards/norik/octopus_io_board/norik_octopus_io_board_nrf9160_ns.dts]: Can we go with:
Copyright (c) 2024 Norik systems d.o.o.
here?

File: boards/norik/octopus_io_board/norik_octopus_io_board_nrf9160_ns.dts [hunk 23]
```diff
@@ -0,0 +1,23 @@
...
+		 zephyr,sram = &sram0_ns;
+		 zephyr,code-partition = &slot0_ns_partition;
+	 };
+ };
+ 
+ /* Disable UART1, because it is used by default in TF-M */
+ &uart1 {
+	 status = "disabled";
+ };
+
```
Comment (pfiser) [inline boards/norik/octopus_io_board/norik_octopus_io_board_nrf9160_ns.dts]: new line here?

File: boards/norik/octopus_io_board/norik_octopus_io_board_nrf9160_ns_defconfig [hunk 24]
```diff
@@ -0,0 +1,4 @@
+# SPDX-License-Identifier: Apache-2.0
+
+# This Board implies building Non-Secure firmware
+CONFIG_TRUSTED_EXECUTION_NONSECURE=y
```
Comment (pfiser) [inline boards/norik/octopus_io_board/norik_octopus_io_board_nrf9160_ns_defconfig]: add new line here

File: boards/norik/octopus_io_board/norik_octopus_io_board_nrf9160_ns_defconfig [hunk 25]
```diff
@@ -0,0 +1,4 @@
+# SPDX-License-Identifier: Apache-2.0
```
Comment (pfiser) [inline boards/norik/octopus_io_board/norik_octopus_io_board_nrf9160_ns_defconfig:1]: Copyright missing or not?

File: boards/norik/octopus_io_board/norik_octopus_io_board_nrf9160_partition_conf.dtsi [hunk 26]
```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) 2023-2024 Norik Systems
```
Comment (pfiser) [inline boards/norik/octopus_io_board/norik_octopus_io_board_nrf9160_partition_conf.dtsi]: Can we go with:
Copyright (c) 2024 Norik systems d.o.o.
here?

File: boards/norik/octopus_som/Kconfig.defconfig [hunk 27]
```diff
@@ -0,0 +1,33 @@
+# Copyright (c) 2023 Nordic Semiconductor ASA
```
Comment (pfiser) [inline boards/norik/octopus_som/Kconfig.defconfig]: Can we go with:
Copyright (c) 2024 Norik systems d.o.o.
here?

File: boards/norik/octopus_som/Kconfig.norik_octopus_som [hunk 28]
```diff
@@ -0,0 +1,5 @@
+# Copyright (c) 2024 Norik Systems
```
Comment (pfiser) [inline boards/norik/octopus_som/Kconfig.norik_octopus_som]: Can we go with:
Copyright (c) 2024 Norik systems d.o.o.
here?

File: boards/norik/octopus_som/board.cmake [hunk 29]
```diff
@@ -0,0 +1,6 @@
+# Copyright (c) 2023 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+board_runner_args(jlink "--device=nRF9160_xxAA" "--speed=4000")
+include(${ZEPHYR_BASE}/boards/common/nrfjprog.board.cmake)
+include(${ZEPHYR_BASE}/boards/common/jlink.board.cmake)
```
Comment (pfiser) [inline boards/norik/octopus_som/board.cmake]: New line here

File: boards/norik/octopus_som/board.cmake [hunk 30]
```diff
@@ -0,0 +1,6 @@
+# Copyright (c) 2023 Nordic Semiconductor ASA
```
Comment (pfiser) [inline boards/norik/octopus_som/board.cmake]: Can we go with:
Copyright (c) 2024 Norik systems d.o.o.
here?

File: boards/norik/octopus_som/doc/index.rst [hunk 31]
```diff
@@ -0,0 +1,163 @@
...
+Pinout Diagram
+--------------
+
+.. figure:: img/Norik_Octopus_Som_pinout.png
+     :align: center
+     :alt: Norik Octopus SoM pinout
+     
+     Norik Octopus SoM pinout
+
+
```
Comment (pfiser) [inline boards/norik/octopus_som/doc/index.rst]: Maybe too much empty lines here?

File: boards/norik/octopus_som/norik_octopus_som_common-pinctrl.dtsi [hunk 32]
```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor
```
Comment (pfiser) [inline boards/norik/octopus_som/norik_octopus_som_common-pinctrl.dtsi]: Use same copyright for all files in this PR?

File: boards/norik/octopus_som/norik_octopus_som_common-pinctrl.dtsi [hunk 33]
```diff
@@ -0,0 +1,67 @@
...
+	};
+
+	uart0_sleep: uart0_sleep {
+		group1 {
+			psels = <NRF_PSEL(UART_TX, 0, 23)>,
+				<NRF_PSEL(UART_RX, 0, 23)>;
+			low-power-enable;
+		};
+	};
+    i2c0_default: i2c0_default {
```
Comment (pfiser) [inline boards/norik/octopus_som/norik_octopus_som_common-pinctrl.dtsi]: identation?

File: boards/norik/octopus_som/norik_octopus_som_common-pinctrl.dtsi [hunk 34]
```diff
@@ -0,0 +1,67 @@
...
+	};
+
+	i2c0_sleep: i2c0_sleep {
+		group1 {
+			psels = <NRF_PSEL(TWIM_SDA, 0, 4)>,
+					<NRF_PSEL(TWIM_SCL, 0, 3)>;
+			low-power-enable;
+		};
+	};
+    i2c1_default: i2c1_default {
```
Comment (pfiser) [inline boards/norik/octopus_som/norik_octopus_som_common-pinctrl.dtsi]: identation

File: boards/norik/octopus_som/norik_octopus_som_common-pinctrl.dtsi [hunk 35]
```diff
@@ -0,0 +1,67 @@
...
+	spi3_sleep: spi3_sleep {
+		group1 {
+			psels = <NRF_PSEL(SPIM_SCK, 0, 10)>,
+				<NRF_PSEL(SPIM_MISO, 0, 8)>,
+				<NRF_PSEL(SPIM_MOSI, 0, 9)>;
+			low-power-enable;
+		};
+	};
+
+};
```
Comment (pfiser) [inline boards/norik/octopus_som/norik_octopus_som_common-pinctrl.dtsi]: new line add

File: boards/norik/octopus_som/norik_octopus_som_common.dtsi [hunk 36]
```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright (c) 2023-2024 Norik Systems
```
Comment (pfiser) [inline boards/norik/octopus_som/norik_octopus_som_common.dtsi]: same copyright

File: boards/norik/octopus_som/norik_octopus_som_common.dtsi [hunk 37]
```diff
@@ -0,0 +1,139 @@
...
+	simselect-controller{
+		compatible = "gpio-keys";
+		sim_select: sim_select {
+			label = "SIM select";
+			gpios = <&gpio0 25 GPIO_ACTIVE_HIGH>;
+		};
+	};
+
+
+   /* These aliases are provided for compatibility with samples */
```
Comment (pfiser) [inline boards/norik/octopus_som/norik_octopus_som_common.dtsi]: identation

File: boards/norik/octopus_som/norik_octopus_som_common.dtsi [hunk 38]
```diff
@@ -0,0 +1,139 @@
...
+   aliases {
+	   led0 = &led0;
+	   pwm-led0 = &pwm_led0;
+	   watchdog0 = &wdt0;
+	   simselect = &sim_select;
+   };
+};
+
+&adc {
```
Comment (pfiser) [inline boards/norik/octopus_som/norik_octopus_som_common.dtsi:48]: identation adc, gpiote, gpio0, uart0 blocks

File: boards/norik/octopus_som/norik_octopus_som_common.dtsi [hunk 39]
```diff
@@ -0,0 +1,139 @@
...
+	cs-gpios = <&gpio0 11 GPIO_ACTIVE_LOW>;
+
+	pinctrl-0 = <&spi3_default>;
+	pinctrl-1 = <&spi3_sleep>;
+	pinctrl-names = "default", "sleep";
+	mc3635: mc3635@0 {
+		compatible = "vnd,spi-device";
+		spi-max-frequency = <1000000>;
+		reg = <0>;
+//		int1-gpios = <&gpio0 12 0>;
```
Comment (pfiser) [inline boards/norik/octopus_som/norik_octopus_som_common.dtsi]: drop if not used?

File: boards/norik/octopus_som/norik_octopus_som_nrf9160.dts [hunk 40]
```diff
@@ -0,0 +1,20 @@
...
+/ {
+	chosen {
+		zephyr,sram = &sram0_s;
+		zephyr,flash = &flash0;
+		zephyr,code-partition = &slot0_partition;
+		zephyr,sram-secure-partition = &sram0_s;
+		zephyr,sram-non-secure-partition = &sram0_ns;
+	};
+};
+
```
Comment (pfiser) [inline boards/norik/octopus_som/norik_octopus_som_nrf9160.dts]: new line

File: boards/norik/octopus_som/norik_octopus_som_nrf9160.yaml [hunk 41]
```diff
@@ -0,0 +1,17 @@
...
+  - zephyr
+  - gnuarmemb
+  - xtools
+supported:
+  - gpio
+  - i2c
+  - spi
+  - pwm
+  - watchdog
+vendor: norik
```
Comment (pfiser) [inline boards/norik/octopus_som/norik_octopus_som_nrf9160.yaml]: new line

File: boards/norik/octopus_som/norik_octopus_som_nrf9160_ns.dts [hunk 42]
```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2023-2024 Norik Systems
```
Comment (pfiser) [inline boards/norik/octopus_som/norik_octopus_som_nrf9160_ns.dts]: unified copyright

File: boards/norik/octopus_som/norik_octopus_som_nrf9160_ns.dts [hunk 43]
```diff
@@ -0,0 +1,23 @@
...
+		 zephyr,sram = &sram0_ns;
+		 zephyr,code-partition = &slot0_ns_partition;
+	 };
+ };
+ 
+ /* Disable UART1, because it is used by default in TF-M */
+ &uart1 {
+	 status = "disabled";
+ };
+
```
Comment (pfiser) [inline boards/norik/octopus_som/norik_octopus_som_nrf9160_ns.dts]: new line

File: boards/norik/octopus_som/norik_octopus_som_nrf9160_ns.yaml [hunk 44]
```diff
@@ -0,0 +1,18 @@
...
+  - gnuarmemb
+  - xtools
+supported:
+  - gpio
+  - i2c
+  - spi
+  - pwm
+  - watchdog
+vendor: norik
+
```
Comment (pfiser) [inline boards/norik/octopus_som/norik_octopus_som_nrf9160_ns.yaml]: new line

File: boards/norik/octopus_som/norik_octopus_som_nrf9160_partition_conf.dtsi [hunk 45]
```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) 2023-2024 Norik Systems
```
Comment (pfiser) [inline boards/norik/octopus_som/norik_octopus_som_nrf9160_partition_conf.dtsi]: unified copyright

**Review by pfiser — CHANGES_REQUESTED** [review]
A lot of nodes in device-tree seem to be duplicated?

Is it possible to define it once and include it later?

For example, why are there two "spi3" nodes duplicated?

File: boards/norik/octopus_io_board/doc/img/Norik_Octopus_IO-Board_front.png
Comment (UKosale) [inline boards/norik/octopus_io_board/doc/img/Norik_Octopus_IO-Board_front.png:1]: The black frame around the image could be trimmed

File: boards/norik/octopus_som/doc/img/Norik Octopus Som back.png
Comment (UKosale) [inline boards/norik/octopus_som/doc/img/Norik Octopus Som back.png:1]: The black frame around the image could be trimmed.

File: boards/norik/octopus_som/doc/img/Norik_Octopus_Som_front.png
Comment (UKosale) [inline boards/norik/octopus_som/doc/img/Norik_Octopus_Som_front.png:1]: The black frame around the image could be trimmed.

File: boards/norik/octopus_io_board/doc/index.rst (hunk 5 above)
Comment (FPlohl) [reply boards/norik/octopus_io_board/doc/index.rst:47]: nRF9160 contains UARTE: https://infocenter.nordicsemi.com/index.jsp?topic=%2Fps_nrf9160%2Fuarte.html 
Though I've labeled it on PCB as just UART

File: boards/norik/octopus_io_board/norik_octopus_io_board_defconfig (hunk 18 above)
Comment (FPlohl) [reply boards/norik/octopus_io_board/norik_octopus_io_board_defconfig]: Others didn't add the copyright in this file and it's really similar if not the same for boards with the same MCU: 

https://github.com/zephyrproject-rtos/zephyr/blob/main/boards/nordic/nrf9160dk/nrf9160dk_nrf9160_defconfig
https://github.com/zephyrproject-rtos/zephyr/blob/main/boards/actinius/icarus_som/actinius_icarus_som_defconfig
https://github.com/zephyrproject-rtos/zephyr/blob/main/boards/circuitdojo/feather/circuitdojo_feather_nrf9160_defconfig

Didn't know if I should add it or not

File: boards/norik/octopus_io_board/norik_octopus_io_board_nrf9160.yaml (hunk 20 above)
Comment (FPlohl) [reply boards/norik/octopus_io_board/norik_octopus_io_board_nrf9160.yaml]: Typo. Should be: norik_octopus_io_board/nrf9160

File: boards/norik/octopus_som/norik_octopus_som_nrf9160.dts (hunk 40 above)
Comment (FPlohl) [reply boards/norik/octopus_som/norik_octopus_som_nrf9160.dts]: Between line 18 and 19?

File: boards/norik/octopus_io_board/norik_octopus_io_board_common.dtsi (hunk 17 above)
Comment (FPlohl) [reply boards/norik/octopus_io_board/norik_octopus_io_board_common.dtsi]: Line 160?

Comment (pfiser) [reply boards/norik/octopus_io_board/norik_octopus_io_board_common.dtsi]: 161

File: boards/norik/octopus_io_board/board.cmake (hunk 3 above)
Comment (FPlohl) [reply boards/norik/octopus_io_board/board.cmake]: Between 4 & 5 or after 6?

Comment (pfiser) [reply boards/norik/octopus_io_board/board.cmake]: after 6

File: boards/norik/octopus_io_board/norik_octopus_io_board_nrf9160_ns.dts (hunk 23 above)
Comment (pfiser) [reply boards/norik/octopus_io_board/norik_octopus_io_board_nrf9160_ns.dts]: I still have red minus sign here :) check again, maybe some hidden char?

File: boards/norik/octopus_som/norik_octopus_som_common.dtsi [hunk 46]
```diff
@@ -0,0 +1,125 @@
...
+	
+	pwmleds {
+		compatible = "pwm-leds";
+
+		pwm_led0: pwm_led_0 {
+			pwms = <&pwm0 0 PWM_MSEC(20) PWM_POLARITY_NORMAL>;
+		};
+	};
+
+	simselect-controller{
```
Comment (pfiser) [inline boards/norik/octopus_som/norik_octopus_som_common.dtsi]: Add space between controller and { bracket.

File: boards/norik/octopus_io_board/norik_octopus_io_board_common.dtsi [hunk 47]
```diff
@@ -0,0 +1,129 @@
...
+		spi-max-frequency = <80000000>;
+		jedec-id = [ef 40 17];
+		size = <0x4000000>;
+		has-dpd;
+		t-enter-dpd = <3500>;
+		t-exit-dpd = <3500>;
+	};
+};
+
+&flash0 {
```
Comment (pfiser) [inline boards/norik/octopus_io_board/norik_octopus_io_board_common.dtsi]: flash0 node seems to be duplicated between Board and SoM.... can we try to get rid of one?

File: boards/norik/octopus_io_board/norik_octopus_io_board_common.dtsi [hunk 48]
```diff
@@ -0,0 +1,129 @@
...
+		};
+		storage_partition: partition@f8000 {
+			label = "storage";
+			reg = <0x000f8000 0x00008000>;
+		};
+	};
+};
+
+/ {
+	reserved-memory {
```
Comment (pfiser) [inline boards/norik/octopus_io_board/norik_octopus_io_board_common.dtsi]: Same here... reserved-memory seems duplicated?

**Comment by FPlohl** [comment]
Requesting review @pfiser @andrejpicej

**Comment by pfiser** [comment]
Can you do before my review:
git fetch origin/main
git rebase main
and then squash commits into logic units

**Review by pfiser — CHANGES_REQUESTED** [review]
We decided to open PR #2 instead. Closing this one.

**Pull request closed without merging**